packages/components/molecules/f-rating/test/visual/f-rating.visual.spec.js

```diff
@@ -32,7 +32,7 @@ devices.forEach(device => {
                 await Rating.load({ starRating: 5 });
 
                 // Assert
-                await browser.percyScreenshot('f-rating - Visual Test for Prop - maxStarRating = 5', device);
+                await browser.percyScreenshot('f-rating Multi Star - Visual Test for Prop - maxStarRating = 5', device);
```

Comment:
Testing `starRating` but prop is `maxStarRating`

Reply:
Good spot on these 👍